ui/onboarding/BeforeGetStartedModal.qml

```diff
@@ -0,0 +1,98 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.14
+
+import "../imports"
+import "../shared"
+import "../shared/status"
+
+ModalPopup {
+    id: popup
+    displayCloseButton: false
+    title: qsTr("Before you get started...")
+    width: 430
+    height: 300
+
+    ColumnLayout {
+        anchors.centerIn: parent
+        width: parent.width
+
+        StatusCheckBox {
+            id: acknowledge
+            Layout.preferredWidth: parent.width
+            text: qsTr("I acknowledge that Status Desktop is in Beta and by using it, I take the"
+                       + "full full responsibility for all risks concerning my data and funds.")
```

Comment:
There is a double "full" on this line. Also, I don't think my translation script works well with `qsTr`s on multiple lines. For the time being, it would be safer to put the whole string on one line, even if it's uglier.